docs/helpers/PrintHelper.md

```diff
@@ -29,6 +30,12 @@ In addition, you can customize the printing dialog using the `PrintHelperOptions
 
 You can call `ShowPrintUIAsync` with a second parameter to determine that the list of controls to print should directly be taken from the content of the container passed to the PrintHelper constructor. In this mode you are responsible for the sizing and the layout.
 
+> [!NOTE] Page breaks are not supported. Every control will be printed on a single page.
```

Comment:
The Page break note is old, needs to be deleted here again, looks like a bad merge.

Reply:
Fixed.